app/backend/src/clj/teet/meeting/meeting_commands.clj

```diff
@@ -161,10 +162,14 @@
                user
                meeting-id
                [{:db/id meeting-id
-                 :meeting/agenda (mapv #(select-keys % [:db/id
-                                                        :meeting.agenda/topic
-                                                        :meeting.agenda/body
-                                                        :meeting.agenda/responsible])
+                 :meeting/agenda (mapv #(merge
+                                         (select-keys % [:db/id
+                                                         :meeting.agenda/topic
+                                                         :meeting.agenda/body
+                                                         :meeting.agenda/responsible])
+                                         (if (string? (:db/id %))
```

Comment:
What's the API contract here exactly? If `:db/id` is string, then we're creating a new one, using a placeholder id I guess. Would be nicer to rather communicate what's happening in terms of domain, e.g. `(new-agenda-item? %)` even if the implementation would only be

```
(defn new-agenda-item? [item]
  (string? (:db/id %)))
```

Reply:
yes, same as in many places... you can create or update with the same